iosApp/fastlane/Fastfile

```diff
@@ -23,10 +23,6 @@ platform :ios do
   desc "Size Demo Project & SDK"
   lane :size_demo_app do
     # Archive App
-    match(
```

Comment:
Can we remove [Matchfile](https://github.com/twilio/twilio-verify-passkeys/blob/main/iosApp/fastlane/Matchfile)?